pvlib/pvsystem.py

```diff
@@ -2878,14 +2878,36 @@ def scale_voltage_current_power(data, voltage=1, current=1):
 
 @renamed_kwarg_warning(
     "0.13.0", "g_poa_effective", "effective_irradiance")
-def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
+def pvwatts_dc(effective_irradiance, temp_cell, pdc0, gamma_pdc, temp_ref=25.,
+               k=0.0, capped_adjustment=False):
     r"""
-    Implements NREL's PVWatts DC power model. The PVWatts DC model [1]_ is:
+    Implements NREL's PVWatts (Version 5) DC power model. The PVWatts Version
+    5 DC model [1]_ is:
 
     .. math::
 
         P_{dc} = \frac{G_{poa eff}}{1000} P_{dc0} ( 1 + \gamma_{pdc} (T_{cell} - T_{ref}))
 
+    This model has also been referred to as the power temperature coefficient
+    model.
+
+    This function accepts an optional irradiance adjustment factor, `k`, based
+    on based on [2]_. This applies a piece-wise adjustment to power based on
+    irradiance, where `k` is the reduction in actual power at 200 W/m^2
+    relative to ideal power calculated linearly from standard test conditions,
+    normalized to nameplate power at standard test conditions.
+
+    .. math::
+
+        k=\frac{0.2P_{dc0}-P_{200}}{P_{dc0}}
+
+    For example, a 500 W module that produces 95 W at 200 W/m^2 (a 5% relative
+    reduction in efficiency) would have a value of `k` = 0.01.
+
```

Comment:
Please include Equation from above with `k`, I'd place it here.

Reply:
Yeah, that comment was not clear. I **was** thinking we should copy the equation for `P_DC` here, and show how `k` modifies the power. But the equation for the adjustment is not simple (the piecewise part), so I retract that thought. If someone wants to know how the adjustment works, we've provided code and the reference.